tests/parametric/test_config_consistency.py

```diff
@@ -118,3 +118,43 @@ def test_specific_env(self, library_env, test_agent, test_library):
 
         span = find_span_in_traces(traces, s1.trace_id, s1.span_id)
         assert span["meta"]["env"] == "dev"
+
+
+@scenarios.parametric
+@features.tracing_configuration_consistency
+class Test_Config_DDTraceAgentURL:
```

Comment:
As a nit: I think removing the DD would fit the convention we've been assigning these names

```suggestion
class Test_Config_TraceAgentURL:
```